### ISSUE TITLE
"Order" column in index showing wrong value

### ISSUE THREAD

**Issue by peteeveleigh** [verbb/gift-voucher#32]
It looks like the "Order" column in the voucher's element index is meant to show the original amount of the voucher with the "Amount" column showing the balance left on that voucher.

However, both values (as shown in the element index) get reduced when a purchase is made using a voucher. In the details page for the voucher the correct original amount is still shown.

For instance;

In the Element Index, a voucher with an original value of £100 will show 

```
Order    |  Amount
--------------------
  £100   |  £100

```

But with a purchase of £20 applied to it, I would expect it to show

```
Order    |  Amount
--------------------
  £100   |  £80

```

But it will instead show

```
Order    |  Amount
--------------------
  £80    |  £80

```

- Plugin version: 2.2.1
- Craft version: 3.4.9

**Comment by engram-design** [comment]
I'll fix that up, Order should certainly not be showing a currency amount, instead a link to the Commerce Order. This would also be for Voucher _Codes_ element index.

**Comment by peteeveleigh** [comment]
Hi, yes sorry it is the Voucher *codes* element index.

The amounts being shown are a result of my import. I set the amount field to be the original total minus the sum of any transactions against that code.

But I can confirm the same behaviour with codes created directly from the CP.

**Comment by engram-design** [comment]
Fixed in [2.2.2](https://github.com/verbb/gift-voucher/releases/tag/2.2.2)
